### PR TITLE
More generic HANA node name in qesap_cluster_logs

### PR DESCRIPTION
**Pull request by mpagot**
Use a more generic Ansible label to target the HANA nodes. Hostname is not generic enough: for example in GCP the host name is composed as vmhana0[1|2]-<resource group name>.

- Related ticket: [TEAM-8869](https://jira.suse.com/browse/TEAM-8869)

# Verification run

## qesap regression
 - sle-15-SP4-Qesap-Gcp-Byos-x86_64-BuildLATEST_GCE_SLE15_4_BYOS-qesap_gcp_sapconf_test@64bit -> http://openqaworker15.qa.suse.cz/tests/273311

Here the change at work http://openqaworker15.qa.suse.cz/tests/273311/logfile?filename=serial_terminal.txt

`ansible-playbook` is called wit the new syntax `hana[0]` and Ansible is properly using it by executing tasks on `qesapval273311-vmhana01`

```
(exec_venv) # ansible-playbook  script_output.yaml -l "hana[0]" -i /root/qe-sap-deployment/terraform/gce/inventory.yaml -u cloudadmin -b --become-user root -e "cmd='lsblk -i -a'" -e out_file='hana0-lsblk.txt' -e remote_path='/tmp/' -e failok=yes; echo cSYBz-$?-

PLAY [all] *********************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************

TASK [Gathering Facts] *********************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
ok: [qesapval273311-vmhana01]

TASK [Run cmd] *****************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
changed: [qesapval273311-vmhana01]

PLAY RECAP *********************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
qesapval273311-vmhana01    : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
```

### CODE REVIEW

File: lib/qesapdeployment.pm [hunk 1]
```diff
@@ -1109,16 +1109,19 @@ sub qesap_cluster_logs {
     my $inventory = qesap_get_inventory(provider => $provider);
     if (script_run("test -e $inventory") == 0)
     {
-        foreach my $host ('vmhana01', 'vmhana02') {
+        foreach my $host ('"hana[0]"', '"hana[1]"') {
```
Comment (alvarocarvajald) [inline lib/qesapdeployment.pm:1112]: Is this a reference to the inventory?

**Review by alvarocarvajald — APPROVED** [review]
LGTM. I have a question though.

File: lib/qesapdeployment.pm (hunk 1 above)
Comment (mpagot) [reply lib/qesapdeployment.pm:1112]: yes it is. This PR is using same strategy we used in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16744

**Comment by lpalovsky** [comment]
LGTM

**Pull request merged**